include/can/core/ids.hpp

```diff
@@ -130,6 +130,8 @@ enum class ErrorCode {
     timeout = 0x6,
     estop_detected = 0x7,
     collision_detected = 0x8,
+    labware_dropped = 0x9,
```

Comment:
ditto

Reply:
this was generated with update-headers from the changes already being in monorepo